Parsers/ASimFileEvent/Parsers/vimFileEventMicrosoftSecurityEvents.yaml

```diff
@@ -0,0 +1,197 @@
+Parser:
```

Comment:
testing failing for this parser
![image](https://github.com/user-attachments/assets/789535a7-4ef3-4833-8233-9d8222b8e86f)